beacon/weakcoin/weak_coin.go

```diff
@@ -49,6 +51,91 @@ type Message struct {
 	Signature []byte
 }
 
+// EncodeScale implements scale codec interface.
+func (t *Message) EncodeScale(enc *scale.Encoder) (total int, err error) {
```

Comment:
why is it implemented manually?

Reply:
implemented manually as discussed with you earlier. Now opened https://github.com/spacemeshos/go-spacemesh/issues/3436 and https://github.com/spacemeshos/go-scale/issues/17 to remove this later.